atsign-foundation/at_app_cli
# Added at_backupkey_flutter sample app

**- What I did**

- Added sample from at_widgets repository
  - at_backupkey_flutter 
- Added json-schemas for brick.yaml and template.yaml to make template maintenance easier

**- How I did it**

- Copied the lib folder from at_widgets repository, wrote the necessary config files
- Built and bundled at_app

**- How to verify it**

1. Pull this branch
2. List the samples with `dart run at_app/bin/at_app.dart create --list-samples`
3. Run `dart run at_app/bin/at_app.dart create -s at_backupkey_flutter <path/to/output>` 
4. Run generated sample app and verify that it work

**- Description for the changelog**
Added at_backupkey_flutter sample app
